experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -311,7 +311,7 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
           if (response.aborted && !response.complete) {
             status = { code: SpanStatusCode.ERROR };
           } else {
-            status = utils.parseResponseStatus(response.statusCode);
+            status = { code: utils.parseResponseStatus(SpanKind.CLIENT, response.statusCode) };
```

Comment:
This feels a bit weird, is there a way to access the span kind on the span itself?

The API type doesn't seem to have it defined so I'm not sure if I'm allowed to use

```ts
status = { code: utils.parseResponseStatus(span.kind, response.statusCode)
```

Which would be more natural here

Reply:
SDK/Sampler may decide to discard this span. As a result the span object you get here is just a dummy instead an instance of the Span class of SDK. This dummy fulfills the API but nothing else.

so in short stay on the API.

Reply:
The API could be used with a third party SDK which defines different public properties and methods. Only those in the API interface are guaranteed to exist at all.